src/perf/deferred-components.md

```diff
@@ -14,34 +14,31 @@ features and assets when needed by the user.
 
 We refer to each uniquely downloadable bundle of Dart
 libraries and assets as a "deferred component".
-This is achieved by using Dart's deferred imports,
-which can be compiled into split AOT shared libraries.
+To load these, use Dart's deferred imports.
```

Comment:
We should link to the page that explains how to use deferred imports: https://dart.dev/language/libraries#lazily-loading-a-library

Reply:
@johnpryan : That page also needs an update based on this issue. It doesn't note Android's capability.